### PR TITLE
“#6Task. Dzmitry Bryt

### PR DESCRIPTION
**Pull request by Daqueze21**
https://travis-ci.org/github/Daqueze21/js-assignments/builds/707906693

### CODE REVIEW

**Comment by FedorovD** [comment]
Please clear PR code not related to Task#6 and click "ready for review" when it will be finished. Thank you!

**Comment by FedorovD** [comment]
image
The PR still contains changes from  Task1-4. Remove them.

**Comment by Daqueze21** [comment]
> image
> The PR still contains changes from Task1-4. Remove them.

Remove solutions from tasks 1-4. My fault!

File: task/03-date-tasks.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ function timeSpanToString(startDate, endDate) {
  *    Date.UTC(2016,3,5,21, 0) => Math.PI/2
  */
 function angleBetweenClockHands(date) {
-    throw new Error('Not implemented');
```
Comment (vramaniuk) [inline task/03-date-tasks.js]: That is redundant change. Please make revert to initial state.

File: task/04-arrays-tasks.js [hunk 2]
```diff
@@ -491,108 +480,104 @@ function distinct(arr) {
...
-    getFalsyValuesCount: getFalsyValuesCount,
-    findAllOccurences: findAllOccurences,
-    sortCitiesArray: sortCitiesArray,
-    getIdentityMatrix: getIdentityMatrix,
-    getIntervalArray: getIntervalArray,
-    distinct: distinct,
-    group: group,
-    selectMany: selectMany,
-    getElementByIndexes: getElementByIndexes,
-    swapHeadAndTail: swapHeadAndTail
```
Comment (vramaniuk) [inline task/04-arrays-tasks.js]: There are redundant changes above too. 
Please make revert to initial state.

**Review by vramaniuk — COMMENTED** [review]
Please, mark the PR as ready for review when it will be ready.

**Comment by Daqueze21** [comment]
Sorry!

**Comment by FedorovD** [comment]
Accepted!

**Pull request closed without merging**